rust/private/rust.bzl

```diff
@@ -354,7 +351,6 @@ rust_library = rule(
                  _rust_library_attrs.items()),
     fragments = ["cpp"],
     host_fragments = ["cpp"],
-    cfg = proc_macro_host_transition,
```

Comment:
Obviously just deleting these lines isn't the solution, but it does get the example to compile.